services/apis/nondbgapUsersync/nonDbgapTasks.js

```diff
@@ -106,36 +108,29 @@ module.exports = {
     return id;
   },
 
-  async presignedURLRequest(fence, project, projectFile, user, token) {
-    console.log(`Creating presigned url with mainAcct user for ${project} after running usersync`);
-    console.log(`${token}`);
-    const signedURLs3 = await fence.do.createSignedUrl(projectFile, ['protocol=s3'], token);
-    if (signedURLs3.status === 200) {
-      console.log(`The presigned url for ${project} files is created. The S3 url -> ${signedURLs3.data.urls}`);
-    }
-    const signedURLgs = await fence.do.createSignedUrl(projectFile, ['protocol=gs'], token);
-    if (signedURLgs.status === 200) {
-      console.log(`The presigned url for ${project} files is created. The GS url -> ${signedURLgs.data.urls}`);
-    }
-
+  async presignedURLRequestS3(fence, project, projectFile, user, token) {
     let projectFileContents3 = null;
-    let projectFileContentgs = null;
-
-    try {
+    console.log(`Creating presigned url with ${user} user for ${project} after running usersync`);
+    const signedURLs3 = await fence.do.createSignedUrl(projectFile, ['protocol=s3'], token);
+    if (signedURLs3.status !== 200) {
+      console.log(`User ${user} with access can not create s3 signed urls and read the file for ${project}`);
```

Comment:
I think it would be helpful to output the request log too. Sometimes, it might not be that the user doesn't have access to it. It might be that something went wrong with indexd record creation and we're getting a 404.

Reply:
I agree, having `request.status` on failure can help debugging.